teuthology/orchestra/opsys.py

```diff
@@ -42,7 +42,7 @@
     "rocky": {
         "8.10": "rocky",
         "9.6": "rocky",
-        "10.0": "rocky",
+        "10": "rocky",
```

Comment:
This change is incorrect and will break provisioning of KVM nodes, we should keep X.Y notation, next version of rocky will be 10.1.